core/controllers/questions_list.py

```diff
@@ -15,6 +15,7 @@
 """Controllers for the questions list in topic editors and skill editors."""
 
 from __future__ import annotations
+from msilib import schema
```

Comment:
Can you please explain the need for this import?

Reply:
I do not know how it came but i removed it now.
Is it okay?